conda/recipes/conda_build_config.yaml

```
@@ -82,6 +82,10 @@ euphonic:
 librdkafka:
   - <=2.1.1
 
+# 1.4.4 causes mantid package build to fail
+rhash:
+  - <=1.4.3
```

If we do `!=` then it will allow the version to go above 1.4.4 that may start working again in the future

```suggestion
  - '!=1.4.4'
```

---

I did consider that, but It would also break if this isn't fixed in the next update. Since the last update to `rhash` was in July, there is a chance that our issue could be caused by an intended change?